app/assets/javascripts/admin/addon/components/dashboard-new-features.gjs

```diff
@@ -1,30 +1,41 @@
 import Component from "@glimmer/component";
 import { tracked } from "@glimmer/tracking";
+import didInsert from "@ember/render-modifiers/modifiers/did-insert";
 import { ajax } from "discourse/lib/ajax";
+import i18n from "discourse-common/helpers/i18n";
+import { bind } from "discourse-common/utils/decorators";
 import DashboardNewFeatureItem from "admin/components/dashboard-new-feature-item";
 
 export default class DashboardNewFeatures extends Component {
   @tracked newFeatures = null;
+  @tracked isLoaded = false;
 
-  constructor() {
-    super(...arguments);
-
-    ajax("/admin/dashboard/new-features.json").then((json) => {
-      if (this.isDestroying || this.isDestroyed) {
-        return;
-      }
-
-      this.newFeatures = json.new_features;
-    });
+  @bind
+  loadNewFeatures() {
+    ajax("/admin/dashboard/new-features.json")
+      .then((json) => {
+        this.newFeatures = json.new_features;
+        this.isLoaded = true;
+      })
+      .catch(() => {
+        this.isLoaded = true;
```

Comment:
It doesn't really matter in this case _but_ usually you would add something like this to `finally`.